src/main/java/io/microprofile/config/spi/ConfigSource.java

```diff
@@ -59,6 +64,19 @@
 	 * @param propertyName the property name
 	 * @return the property value 
 	 */
-	String getProperty(String propertyName);
+	String getPropertyValue(String propertyName);
+	
+	/**
+     * The name of the config might be used for logging or analysis of configured values.
+     *
+     * @return the 'name' of the configuration source, e.g. 'property-file mylocation/myproperty.properties'
```

Comment:
How about just get()? Isn't Property redundant (and potentially confusing given environment variables are not usually considered properties?)?